src/common/components/share/styles.css

```diff
@@ -18,8 +18,12 @@
 }
 
 .clipboard {
-  background-image: url(https://assets.ubuntu.com/v1/9cbcbbc7-copy-clipboard.svg);
-  background-size: 12px;
+  background: transparent;
```

Comment:
As commented separately - we need design review for how clipboard should look in share icons.